e2e-tests/portal/cypress/support/portal/loginGoToDealPage.js

```diff
@@ -1,6 +1,11 @@
-const pages = require('../../e2e/pages');
-
-module.exports = (user, deal) => {
+/**
+ * loginGoToDealPage
+ * Login and go to the deal page
+ * @param {Object} user
+ */
+const loginGoToDealPage = (user) => {
   cy.login(user);
-  pages.contract.visit(deal);
+  cy.clickDashboardDealLink();
 };
```

Comment:
I dont think this is a particularly useful command - its unclear what deal page is being navigated to

Reply:
@ttbarnes please advise?

Reply:
What's the page actually called, is it "deal summary" We could add this to the title so it's clearer.

Without this command we'll have the same 2x lines of codes and `pages` imported when it's not necessary.

Reply:
To be checked.

Reply:
@ttbarnes -sorry it's not clear. Should I update the tests with 'deal summary' as there are ~17 files using clickDashboardDealLink()